vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentListComponent.java

```diff
@@ -0,0 +1,193 @@
+import life.qbic.datamanager.views.support.experiment.ExperimentItem;
+import life.qbic.datamanager.views.support.experiment.ExperimentItemClickedEvent;
+import life.qbic.datamanager.views.support.experiment.ExperimentItemCollection;
+import life.qbic.projectmanagement.application.AddExperimentToProjectService;
+import life.qbic.projectmanagement.application.ExperimentInformationService;
+import life.qbic.projectmanagement.application.ExperimentalDesignSearchService;
+import life.qbic.projectmanagement.domain.project.ProjectId;
+import life.qbic.projectmanagement.domain.project.experiment.Experiment;
+import life.qbic.projectmanagement.domain.project.experiment.ExperimentId;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Experiment list component
+ * <p>
+ * The list component is a {@link PageArea} component, which is responsible for showing the
+ * {@link Experiment} information for all experiments in its {@link ExperimentItemCollection }
+ * within the currently examined {@link life.qbic.projectmanagement.domain.project.Project}.
+ * Additionally, it provides the possibility to create new experiments with its
+ * {@link ExperimentCreationDialog} and enables the user to select an experiment of interest via
+ * clicking on the {@link ExperimentItem} associated with the experiment. Finally, it allows
+ * components to be informed about a new experiment creation or selection via the
+ * {@link ExperimentCreationListener} and {@link ExperimentSelectionListener} provided in this
+ * component.
+ */
+@SpringComponent
+@UIScope
+public class ExperimentListComponent extends PageArea {
+
+  @Serial
+  private static final long serialVersionUID = -2196400941684042549L;
+  private final ExperimentItemCollection experimentItemCollection;
+  private ProjectId projectId;
+  private final List<ExperimentSelectionListener> selectionListeners = new ArrayList<>();
+  private final List<ExperimentCreationListener> creationListener = new ArrayList<>();
+  private final ExperimentCreationDialog experimentCreationDialog;
+  private final transient AddExperimentToProjectService addExperimentToProjectService;
+
+  public ExperimentListComponent(
+      @Autowired ExperimentInformationService experimentInformationService,
+      @Autowired AddExperimentToProjectService addExperimentToProjectService,
+      @Autowired ExperimentalDesignSearchService experimentalDesignSearchService) {
+    Objects.requireNonNull(experimentInformationService);
+    Objects.requireNonNull(addExperimentToProjectService);
+    Objects.requireNonNull(experimentalDesignSearchService);
+    this.addExperimentToProjectService = addExperimentToProjectService;
+    this.experimentCreationDialog = new ExperimentCreationDialog(experimentalDesignSearchService);
+    this.addClassName("list-component");
+    this.experimentItemCollection = ExperimentItemCollection.create(
+        "Add a new experiment");
+    this.add(experimentItemCollection);
+    addListeners();
```

Comment:
I introduced methods in the constructor like `layoutComponent()` and `configureComponent()` to make the navigation through components more easy and standardized.

Reply:
done 👍